compiler/quilt/tools/command.py

```diff
@@ -532,7 +532,7 @@ def _build_internal(package, path, dry_run, env, build_file):
                     rmtree(tmpdir)
         else:
             build_from_path(package, path, dry_run=dry_run, env=env)
-    elif isinstance(path, nodes.GroupNode):
+    elif isinstance(path, (nodes.GroupNode, nodes.DataNode)):
```

Comment:
Actually, you can just check for `nodes.Node` -  it's their parent class, and there are no other subclasses.

Reply:
Will do!